nevergrad/optimization/differentialevolution.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, dimension: int, budget: Optional[int] = None, num_workers: in
         self._lhs_init = False
         self._qr_init = False
         self._por_DE = False
+        self._recommendation = "optimistic"
```

Comment:
maybe this should be pessimistic.
In most cases this does not change anything (DE is continuous, all points are distinct) but it's cleaner (intuitively recommendations are pessimistic).

Reply:
Changed to pessimistic

Reply:
Reverted to optimistic because pessimistic deteriorates results